### PR TITLE
wrote a test case which change to_anf considering issue 26985

### PR DESCRIPTION
**Pull request by bharatAmeria**
#### Release Notes

NO ENTRY

### CODE REVIEW

File: sympy/logic/tests/test_boolalg.py [hunk 1]
```diff
@@ -1341,12 +1341,28 @@ def test_relational_threeterm_simplification_patterns_numerically():
...
+    a, b, c, d = symbols('a b c d')
+
+    # Expression before applying to_anf
+    x = Xor(c, And(a, b), And(a, c))
+    y = Xor(a, b, And(a, c))
+
+    # Applying to_anf
+    result = Xor(Xor(d, And(x, y)), And(x, y))
+    result_anf = to_anf(Xor(to_anf(Xor(d, And(x, y))), And(x, y)))
+    expected_result_anf = a * c ^ a * b * c ^ b * c ^ b ^ a * d ^ a * c * d ^ a * b * c * d ^ b * c * d ^ c * d ^ a
```
Comment (smichr) [inline sympy/logic/tests/test_boolalg.py:1365]:
```suggestion
    expected_result_anf = a*c^a*b*c^b*c^b^a*d^a*c*d^a*b*c*d^b*c*d^c*d^a
```

**Comment by smichr** [comment]
looks like you will need to add yourself to the AUTHORS file. See [here](https://github.com/sympy/sympy/actions/runs/10617628219/job/29431048448?pr=27017)

**Comment by oscarbenjamin** [comment]
> the AUTHORS file.

Not the AUTHORS file. You should add an entry in the .mailmap file.

**Comment by oscarbenjamin** [comment]
It looks like your editor has used an auto-formatter that has changed many unrelated lines of code. Please disable the auto-formatter and do not change the unrelated lines of code because it makes it difficult to see what the actual changes are when looking the diff (See the "files changed" tab above).

**Comment by bharatAmeria** [comment]
thank you @oscarbenjamin for guiding me. I am new to this project. I updated .mailmap file by adding name and email but the checks are failing can you guide what should I do

**Comment by bharatAmeria** [comment]
I really apologise @oscarbenjamin for the auto formatter thing. Now I disable it. In future I'll keep in mind and check for the difference and changes in file

**Comment by sylee957** [comment]
I think that auto formatter result is okay though, although we warn about novice usage,
it may be suggested for improve the workflow, and no need to revert the changes,
but you need to fix the authors file

File: sympy/logic/tests/test_boolalg.py [hunk 2]
```diff
@@ -1341,12 +1341,27 @@ def test_relational_threeterm_simplification_patterns_numerically():
...
+    # Expression before applying to_anf
+    x = Xor(c, And(a, b), And(a, c))
+    y = Xor(a, b, And(a, c))
+
+    # Applying to_anf
+    result = Xor(Xor(d, And(x, y)), And(x, y))
+    result_anf = to_anf(Xor(to_anf(Xor(d, And(x, y))), And(x, y)))
+
+    assert result_anf
+    assert result
```
Comment (sylee957) [inline sympy/logic/tests/test_boolalg.py:1367]: The assert statements looks like incomplete
```suggestion
    assert result_anf == d
    assert result == d
```

**Comment by oscarbenjamin** [comment]
The OP here should have had a reference to the associated issue gh-26985 and in particular it should have said "closes gh-26985" so that the issue would be closed automatically when it gets merged.

File: .mailmap [hunk 3]
```diff
@@ -1582,6 +1582,7 @@ anutosh491 <andersonbhat491@gmail.com> Anutosh Bhat <87052487+anutosh491@users.n
 anutosh491 <andersonbhat491@gmail.com> anutosh491 <87052487+anutosh491@users.noreply.github.com>
 arooshiverma <av22@iitbbs.ac.in>
 atharvParlikar <atharvparlikar@gmail.com>
+bharatAmeria <21001019007@jcboseust.ac.in> bharatAmeria <147488804+bharatAmeria@users.noreply.github.com>
```
Comment (oscarbenjamin) [inline .mailmap:1585]: This should be your real name like:
```
Bharat Ameria <email address> bharatAmeria <21001019007@jcboseust.ac.in> bharatAmeria <147488804+bharatAmeria@users.noreply.github.com>
```
@smichr before merging please ask someone whether they actually want this name to be the one that gets recorded in the AUTHORS file.

@bharatAmeria is this the name that you would to have recorded in the AUTHORS file?

Comment (bharatAmeria) [reply .mailmap:1585]: Bharat Ameria this is my real name. Should, I change this name

**Pull request merged**